index.bs

```diff
@@ -308,15 +294,103 @@ The {{Blob()}} constructor can be invoked with the parameters below:
   <dd>which takes any number of the following types of elements, and in any order:
     * {{BufferSource}} elements.
     * {{Blob}} elements.
-    * {{DOMString}} elements.
+    * {{USVString}} elements.
 
   <dt id="dfn-BlobPropertyBagMembers">An *optional* {{BlobPropertyBag}}
-  <dd>which takes one member:
+  <dd>which takes two members:
     * <dfn id="dfn-BPtype" dict-member for="BlobPropertyBag">type</dfn>,
       the ASCII-encoded string in lower case representing the media type of the {{Blob}}.
       Normative conditions for this member are provided in the [[#constructorBlob]].
+    * <dfn dict-member for="BlobPropertyBag">endings</dfn>,
+      an enum which can take the values {{"transparent"}} or {{"native"}}.
+      By default this is set to {{"transparent"}}. If set to {{"native"}},
+      [=convert line endings to native|line endings will be converted to native=]
+      in any {{USVString}} elements in {{blobParts}}.
 </dl>
 
+<div algorithm="process-blob-parts">
+To <dfn lt="process blob parts|processing blob parts">process blob parts</dfn> given a sequence of {{BlobPart}}'s |parts|
+and {{BlobPropertyBag}} |options|,
+run the following steps:
+
+1. Let |bytes| be an empty sequence of bytes.
+
+1. For each |element| in |parts|:
+
+  1. If |element| is a {{USVString}}, run the following substeps:
+
+    1. Let |s| be |element|.
+
+    1. If the {{BlobPropertyBag/endings}} member of |options| is set to {{"native"}},
```

Comment:
nit: I think "set" is a verb in spec language, so "is set to" is used during initialization, not testing. Could just be `|options| is {{"native"}}` here.

Reply:
Done